Controller/Callback/Offsite.php

```diff
@@ -77,7 +78,7 @@ public function execute()
             return $this->redirect(self::PATH_CART);
         }
 
-        if (! in_array($payment->getMethod(), [Alipay::CODE, Internetbanking::CODE])) {
+        if (! in_array($payment->getMethod(), [Alipay::CODE, Internetbanking::CODE, Installment::CODE])) {
```

Comment:
@jacstn at about the line #131 of this file still saying 
`'Amount of %1 has been paid via Omise Internet Banking payment',`
(in case of successful charge).